packages/flutter/lib/src/widgets/snapshot_widget.dart

```diff
@@ -0,0 +1,434 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter/rendering.dart';
+
+import 'basic.dart';
+import 'debug.dart';
+import 'framework.dart';
+import 'media_query.dart';
+
+/// Controls how the [SnapshotWidget] paints its child.
+enum SnapshotMode {
+  /// The child is snapshotted, but only if all descendants can be snapshotted.
+  ///
+  /// If there is a platform view in the children of a raster widget, the
```

Comment:
nit: its not called raster widget anymore..

Reply:
Done